app/server/tests/test_custom_auth.py

```diff
@@ -0,0 +1,16 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase, override_settings
+
+from .. import custom_auth
+
+@override_settings(HEADER_AUTH_PROXY_GROUPS='Admin')
+@override_settings(HTTP_X_AUTH_PROXY_USER='UserA')
+@override_settings(STATICFILES_STORAGE='django.contrib.staticfiles.storage.StaticFilesStorage')
+class CustomSSOMiddlewareTest(TestCase):
+
+    header = 'HTTP_X_AUTH_PROXY_USER'
```

Comment:
Should use `@override_settings` to ensure that the setting is the same as this header value.